psi4/src/psi4/libtrans/integraltransform_sort_mo_tpdm.cc

```diff
@@ -40,6 +40,20 @@
 
 using namespace psi;
 
+
+size_t *init_size_t_array(int size) {
+    size_t *array;
+
+    if ((array = (size_t *)malloc(sizeof(size_t) * size)) == nullptr) {
+        outfile->Printf("init_size_t_array:  trouble allocating memory \n");
+        outfile->Printf("size = %d\n", size);
+        exit(PSI_RETURN_FAILURE);
+    }
+    memset(array, 0, sizeof(size_t) * size);
+    return (array);
+}
```

Comment:
should this be in libciomr.h with its friends?

Reply:
Yeah, I'll do it.  Might be my first contribution to ye old libciomr.  Can't recall.  But don't ask me to change init_int_array()!  I don't want to know how many little ways it might be used.